src/examples/PopupExamplePage.tsx

```diff
@@ -0,0 +1,319 @@
+        setShowPopup4(false);
+      }}>
+      <Text>Close Popup</Text>
+    </TouchableHighlight>
+  </View>
+</Popup>`;
+
+  return (
+    <Page
+      title="Popup"
+      description="Displays a dismissable view of content on top of existing content in the application window."
+      wrappedNativeControl={{
+        control: 'Popup',
+        url:
+          'https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.controls.primitives.popup?view=winrt-19041',
+      }}
+      pageCodeUrl="https://github.com/microsoft/react-native-gallery/blob/main/src/examples/PopupExamplePage.tsx"
+      documentation={[
+        {
+          label: 'Popup',
+          url:
+            'https://microsoft.github.io/react-native-windows/docs/popup-component',
+        },
+        {
+          label: 'Popup Source Code',
+          url:
+            'https://github.com/microsoft/react-native-windows/blob/master/vnext/Microsoft.ReactNative/Views/PopupViewManager.h',
+        },
+      ]}>
+      <Example title="A simple Popup." code={example1jsx}>
+        <TouchableHighlight
+          style={{
+            height: 40,
+            width: 150,
+            backgroundColor: '#E6E6E6',
+            borderRadius: 3,
+            justifyContent: 'center',
+            alignItems: 'center',
+          }}
+          onPress={() => {
+            setShowPopup1(true);
+          }}>
+          <Text>Open Popup</Text>
+        </TouchableHighlight>
+        <Popup isOpen={showPopup1}>
+          <View
+            style={{
+              backgroundColor: 'white',
+              width: 200,
+              height: 125,
+              borderRadius: 3,
+              alignItems: 'center',
+              justifyContent: 'space-around',
+            }}>
+            <Text>This is a popup.</Text>
+            <TouchableHighlight
+              style={{
+                height: 40,
+                width: 150,
+                backgroundColor: '#E6E6E6',
+                borderRadius: 3,
+                justifyContent: 'center',
+                alignItems: 'center',
+              }}
+              onPress={() => {
+                setShowPopup1(false);
+              }}>
+              <Text>Close Popup</Text>
+            </TouchableHighlight>
+          </View>
+        </Popup>
+      </Example>
+      <Example
+        title="A Popup that is dismissed when user clicks away."
+        code={example2jsx}>
+        <TouchableHighlight
+          style={{
+            height: 40,
+            width: 150,
+            backgroundColor: '#E6E6E6',
+            borderRadius: 3,
+            justifyContent: 'center',
+            alignItems: 'center',
+          }}
+          onPress={() => {
+            setShowPopup2(true);
+          }}>
+          <Text>Open Popup</Text>
+        </TouchableHighlight>
+        <Popup
+          isOpen={showPopup2}
+          isLightDismissEnabled={true}
+          onDismiss={() => {
+            setShowPopup2(false);
+          }}>
```

Comment:
Since isLightDismissEnabled == true, do we actually need an onDismiss handler to hide the Popup?

Reply:
I gave it a try without the onDismiss and it seems to glitch because the state variable controlling when to show the pop up is still set to true even though the popup was dismissed. When the user tries to open the popup again, no popup appears.